### ISSUE TITLE
FrameWork

### ISSUE THREAD

**Issue by EngDinaSayed** [EngDinaSayed/Software-Engineering---Phase-3#2]
An error message appears whenever trying to apply solid as a framework which maybe because of RequestMapping or ExceptionHandler.

**Comment by Soo-12** [comment]
Yes , this error message appears a lot . It is hard to decide the reason for this error.

**Comment by EngDinaSayed** [comment]
Probably, it's almost solved!
